lms/templates/discussion/index.html

```diff
@@ -35,9 +35,9 @@
          data-user-cohort-id="${user_cohort}"
          data-course-settings="${course_settings}">
     <div class="discussion-body">
-        <div class="forum-nav"></div>
-        <div class="discussion-column">
-          <article class="new-post-article" style="display: none"></article>
+        <div class="forum-nav" role="complimentary" aria-label="Discussion thread list"></div>
+        <div class="discussion-column" role="complementary" aria-label="Active Discussion" aria-live="assertive" id="discussion-column">
```

Comment:
I thought you suggested using `assertive` but maybe that was an earlier suggestion. I hardly ever use it for those reasons. I updated this to `polite` but am wondering if a live region is needed at all here since focus goes first to the loading indicator and is then sent back to the main content container.